l10n_fr_siret/models/res_partner.py

```diff
@@ -1,21 +1,47 @@
-import logging
+from stdnum.fr.siren import is_valid as siren_is_valid
+from stdnum.fr.siret import is_valid as siret_is_valid
 
 from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
 
-logger = logging.getLogger(__name__)
-try:
```

Comment:
FTR, encapsulating imports in `try` blocks is not longer necessary in recent Odoo versions.

Odoo won't import these `py` files unless the module is installed, and it can't be installed without the library: https://github.com/OCA/l10n-france/blob/c3208b5183dff5232988c376c2595f855b1751a8/l10n_fr_siret/__manifest__.py#L15